packages/kit/test/apps/basics/package.json

```diff
@@ -9,7 +9,10 @@
 		"check": "svelte-kit sync && tsc && svelte-check",
 		"test": "node test/setup.js && pnpm test:dev && pnpm test:build",
 		"test:dev": "rimraf test/errors.json && cross-env DEV=true playwright test",
-		"test:build": "rimraf test/errors.json && playwright test"
+		"test:build": "rimraf test/errors.json && playwright test",
```

Comment:
I notice that `writes/package.json` also does `rimraf test/errors.json` despite it only being created in the `basics` test, so perhaps we could clean remove it from that file

Reply:
done. would love to find a way to not need that `errors.json` thing, it's super awkward

Reply:
i wonder if we could put the error on `event.locals` then return it with the response via a `x-error` header or something. might try that after we merge this change